Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -92,7 +93,8 @@ def print_filename(string, filename)
       next unless ARGV.verbose? && !text_matches.empty?
       print_filename string, file
       text_matches.first(MAXIMUM_STRING_MATCHES).each do |match, offset|
-        puts " #{Tty.gray}-->#{Tty.reset} match '#{match}' at offset #{Tty.em}0x#{offset}#{Tty.reset}"
+        puts " #{Tty.gray}-->#{Tty.reset} match '#{match}' at offset #{Tty.underline}0x#{offset}#{Tty.reset}"
+
```

Comment:
Also, in this specific case I would prefer to swap the underline for something else. `Tty.bold` would be fine by me, but I'm definitely open to alternative suggestions. It's just that the underline seriously and unnecessarily degrades legibility of those hex numbers.